decArgo_api/decoder_bindings/decoder.py

```diff
@@ -0,0 +1,102 @@
+"""Decoder Bindings."""
+
+import logging
+import os
+import subprocess
+import traceback
+from pathlib import Path
+
+from pydantic import BaseModel, field_validator
+
+
+class EmptyInputDirectoryError(Exception):
+    """Raised when the input directory is empty."""
+
+
+class ExecutionError(Exception):
+    """Raised when no wmonum is passed."""
```

Comment:
This doesn't mean much to me at a glance. Can it be more descriptive? Should it inherit from something more specific, like `KeyError` or `ValueError`?